configure.ac

```diff
@@ -1432,7 +1432,7 @@ if test x$need_bundled_univalue = xyes; then
   AC_CONFIG_SUBDIRS([src/univalue])
 fi
 
-ac_configure_args="${ac_configure_args} --disable-shared --with-pic --with-bignum=no --enable-module-recovery --disable-jni"
+ac_configure_args="${ac_configure_args} --disable-shared --with-pic --with-bignum=no --enable-module-recovery --enable-experimental --enable-module-ecdh --disable-jni"
```

Comment:
Is this `--enable-experimental` "permanent"?

Reply:
It's an upstream question. Should be asked at https://github.com/bitcoin-core/secp256k1

Reply:
It's declared experimental upstream. Asking upstream to no longer declare it experimental is an upstream question. My point is about using it here; as long as it's experimental upstream, shouldn't it be considered experimental here?

Reply:
I think it's best to not hide it behind a configuration option on our side. I think we should always compile BIP324 features but only activate them when passing a startup argument (can be switched to default later)

Reply:
Hallo saya punya tambang bitcoin cuma ngak bisa di ambil Karan email nya syang ngak bisa login.dan tambang coinbase yang ada di dalam emai saya apa bisa di ambil..cuma alamat coinbase tambang nya ada di tsama saya...apa kah bisa sya cek coinbese saya menguna kan alamat nya. Itu menjadi kendala untuk saya. Coin yang ada di dalam tambang apa kah sya bisa jual menguna kan alamat nya yang ada sam saya...tolong bantuan nya rekan rekan..terimakasi.